pol.html

```diff
@@ -348,10 +348,9 @@ <h2 id="milestones">Set Conformance Milestones</h2>
 
 		<p class="totop"><a href="#contents">back to page contents</a></p>
 
-
 		<h2 id="thirdparty">Consider Third-party Content</h2>
-		<p>Procured or syndicated third-party content needs to be considered as part of an organizational policy. Where there is a separate procurement policy or process, include the accessibility policy standards and conformance levels as requirements that third-party suppliers must meet. Also consider what verification tasks, such as expert evaluation and/or user testing, should be included in the procurement process.</p>
-		<p>Syndicated content may not be obtained as part of a procurement contract, for example, a social media feed. This content may need to be moderated or modified to ensure it meets the organizational accessibility policy.</p>
+		<p>Procured or syndicated third-party content needs to be considered as part of your accessibility policy. This is particularly relevant for third-party content that provides essential parts of a website. For example, this could include credit card payment services, video streaming channels, and online mapping services that consitute website content and functionality.</p>
+		<p>While website owners may not have full control over such content and services, they are responsible for ensuring accessibility or for providing accessible alternatives. For example, social media feeds may need to be monitored or moderated to ensure accessibility. See also the <a href="https://www.w3.org/TR/WCAG20/#cc3">WCAG 2.0 Conformance Requirement for "Complete processes"</a>.</p>
```

Comment:
This is the only place where we point to specific WCAG conformance requirements - bug or feature? It feels odd all alone. I suggest adding others as well but don't know where and how. Maybe in the "scope of policy" section and in the new "mobile content and applications" section, if we add it?